padevents/padevents.py

```diff
@@ -679,6 +683,24 @@ async def aed_e_searchstr(self, ctx, index, *, searchstr):
             dmevents[index - 1]['searchstr'] = searchstr
         await ctx.tick()
 
+    @aed_e.command(name="toggle3p")
+    async def aed_e_toggle3p(self, ctx, index: int):
+        """Include/exclude 3-player dungeons in an autoeventdm"""
+        async with self.config.user(ctx.author).dmevents() as dmevents:
+            if not 0 < index <= len(dmevents):
+                await ctx.send("That isn't a valid index.")
```

Comment:
Can we start using the tsutils send_cancellation_message & send_confirmation_message for error/confirmations?

so send a cancellation message here cos it's a no-op/error, and then later on, send a confirmation error because it worked!